.travis.yml

```diff
@@ -2,16 +2,27 @@ language: python
 
 python:
   - 3.6
+
+services:
+  - postgresql
+
+before_script:
+- psql -c "CREATE DATABASE tch3;" -U postgres
+- psql -c "CREATE USER kwanj WITH PASSWORD '4084theG';" -U postgres
+
 install:
   - pip install -r requirements.txt
-
   - pip install coveralls
 env:
   - FLASK_VERSION=1.0.2
 
 script: 
+  - export TEST_URL="dbname=tch3 user=kwanj password=4084theG"
   - pytest --cov-report term-missing --cov=app
   - coveralls
 
 after_success:
-  - coveralls
\ No newline at end of file
+  - coveralls
+
+cache:
+  - pip
```

Comment:
At the end of the file please leave one empty line.

Reply:
I will